core/src/main/scala/org/broadinstitute/dsde/rawls/workspace/WorkspaceService.scala

```diff
@@ -2146,6 +2157,19 @@ class WorkspaceService(protected val userInfo: UserInfo, val dataSource: SlickDa
       }
     }
   }
+
+  private def submissionStatusCounter(workspaceName: WorkspaceName, status: SubmissionStatus): Counter =
+    ExpandedMetricBuilder
+      .expand(WorkspaceMetric, workspaceName)
+      .expand(SubmissionStatusMetric, status)
```

Comment:
Actually I was intending to count submission statuses per workspace, so it is not scoped to a single submission.